src/include/cursor.h

```diff
@@ -413,7 +416,9 @@ struct __wt_cursor_log {
 	uint32_t	step_count;	/* Intra-record count */
 	uint32_t	rectype;	/* Record type */
 	uint64_t	txnid;		/* Record txnid */
-	uint32_t	flags;
+
+#define	WT_CURLOG_ARCHIVE_LOCK	0x01	/* Archive lock held */
+	u_int		flags;
```

Comment:
`flags` are always sized and we have a 64-bit version.  I'd prefer these be/remain `uint32_t`.